src/analysis/complex/exponential.lean

```diff
@@ -120,8 +124,8 @@ end
 lemma differentiable_cos : differentiable ℂ cos :=
 λx, (has_deriv_at_cos x).differentiable_at
 
-@[simp] lemma deriv_cos {x : ℂ} : deriv cos x = -sin x :=
-(has_deriv_at_cos x).deriv
+@[simp] lemma deriv_cos : deriv cos = (λ x, -sin x) :=
```

Comment:
Why do you choose to make `deriv_id` and `deriv_id'`, but only `deriv_cos` and not `deriv_cos'`?

Reply:
This is one of the reasons for marking this as a WIP. First I tried to stick to lemmas with functional equality only but then one can't do `rw [← deriv_id]`. So I restored some of the old lemmas. Now I think whether I should define two lemmas for each function, or only for those where RHS for the `funext` version uses `λ _, _`.

Reply:
Fixed.